src/main.rs

```diff
@@ -56,6 +57,9 @@ fn try_main() -> Result<(), ProgramError> {
         .arg(Arg::with_name("yolo")
             .long("yolo")
             .help("Automatically apply all unambiguous suggestions"))
+        .arg(Arg::with_name("apply-only-use")
+             .long("apply-only-use")
+             .help("Apply only use fix suggestions"))
```

Comment:
I'd rather we introduce an `--apply` or `--enable` or `--only` flag (can be used multiple times) and, when set, rustfix will filter the given suggestions to only fix these lints. 

For now this can be hardcoded to only accept "use" (or whatever the actual lint name is) to keep the code simple.